plugins/node/opentelemetry-plugin-ioredis/package.json

```
@@ -48,7 +48,7 @@
     "@opentelemetry/node": "0.8.3",
     "@opentelemetry/test-utils": "^0.8.0",
     "@opentelemetry/tracing": "0.8.3",
-    "@types/ioredis": "4.16.5",
+    "@types/ioredis": "^4.16.6",
```

Now that we are using dependabot, we are pinning versions.
```suggestion
    "@types/ioredis": "4.16.6",
```